src/Reflection/Abstraction.agda

```diff
@@ -19,7 +20,8 @@ open import Relation.Binary.PropositionalEquality
 
 private
   variable
-    A B : Set
+    a b : Level
+    A B : Set a
```

Comment:
These changes seem unnecessary?

Reply:
Without these changes, the function `≡-dec` below is not level-polymorphic, but I need the polymorphic version in order to define the instance for `Abs` in `Reflection.Instances`.